### PR TITLE
test: add coverage on create-llama

### PR DESCRIPTION
**Pull request by himself65**
Refs: https://github.com/run-llama/LlamaIndexTS/pull/231#issuecomment-1825240140

This is different with vercel way. They compare the output file to see if it's correct. But we did run the server to see if it's working.

### CODE REVIEW

File: packages/create-llama/e2e/basic.spec.ts [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+test("try create-llama", async ({ page }) => {
+  const createLlama = fileURLToPath(
+    new URL("../dist/index.js", import.meta.url),
+  );
+
+  const name = "basic";
+  const template: TemplateType = "streaming";
+  const framework: TemplateFramework = "nextjs";
+  const engine: TemplateEngine = "simple";
+  const ui: TemplateUI = "shadcn";
```
Comment (marcusschiesser) [inline packages/create-llama/e2e/basic.spec.ts:32]: @himself65 so currently we're testing only this configuration right? How about adding other permutations?

Comment (himself65) [reply packages/create-llama/e2e/basic.spec.ts:32]: I will add them later

**Comment by himself65** [comment]
I think this E2E is too costly since all enumeration will be generated. Thinking to improve this later

**Comment by marcusschiesser** [comment]
> I think this E2E is too costly since all enumeration will be generated. Thinking to improve this later
Some ideas:
- These tests only need to run for create-llama - move it to its own repo? (So changes on LITS are not triggering this e2e)
- Run each test permutation in its own process

**Comment by marcusschiesser** [comment]
closing, will open a new one

**Pull request closed without merging**